starter-code/scripts/articleView.js

```diff
@@ -19,48 +19,73 @@ articleView.populateFilters = function() {
 articleView.handleAuthorFilter = function() {
   $('#author-filter').on('change', function() {
     if ($(this).val()) {
-      /* TODO: If the select box changes to an option that has a value, we should:
+      /* TODONE: If the select box changes to an option that has a value, we should:
           1. Hide all of the articles
           2. Fade in only the articles that match based on on the author
             that was aselected. Hint: use an attribute selector to find
             those articles that match the value, and then fade them in.
         */
+      $('article').hide();
+      $('article[data-author-name="' + $(this).val() + '"]').fadeIn('slow');
+
     } else {
-    /* Otherwise, we should:
-        1. Show all the articles except the template */
+      $('article').show();
+      $('article.template').hide();
```

Comment:
Awesome! Another way to show all the articles except template is this: $('article').not('template').show();